examples/src/main/scala/KleisliExample.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2022 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import cats.data.Kleisli
+import cats.effect.IO
+import cats.effect.IOApp
+import cats.effect.Resource
+import cats.effect.Sync
+import cats.mtl.Local
+import io.opentelemetry.api.GlobalOpenTelemetry
+import org.typelevel.otel4s.java.trace.Scope
+import org.typelevel.otel4s.java.trace.Traces
+import org.typelevel.otel4s.trace.Tracer
+
+object KleisliExample extends IOApp.Simple {
+  def work[F[_]: Sync: Tracer] =
+    Tracer[F].span("work").surround(Sync[F].delay(println("I'm working")))
+
+  def tracerResource[F[_]](implicit
+      F: Sync[F],
+      L: Local[F, Scope]
+  ): Resource[F, Tracer[F]] =
+    Resource
+      .eval(Sync[F].delay(GlobalOpenTelemetry.get))
+      .map(Traces.local[F])
```

Comment:
We could thread `Local` all the way through to `OtelJava` and make it look more like the other example?  Everything would run through a `Scope`, including metrics...